src/parsers/cs2.py

```
@@ -0,0 +1,130 @@
+"""
+Counter-Strike 2 parser adapter implementing the PandaScoreParser
+interface. This is a conservative implementation based on expected
+PandaScore payloads and the same contract used by the LoL parser.
+"""
+
+import logging
+from typing import Optional, Dict, Any, Tuple
```

Ruff (0.15.5)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)